lib/cocoapods/generator/embed_frameworks_script.rb

```diff
@@ -45,6 +45,15 @@ def script
 
           install_framework()
           {
+            if [ -e "$1" ]; then
+              version=$(defaults read "${PODS_ROOT}/Target Support Files/$(basename -s .framework "$1")/Info.plist" CFBundleShortVersionString)
```

Comment:
@dnkoutso Exactly what I was looking for! :) 👍 Great job and thanks! Sorry again for missing out on checking this beta version. Will be looking forward to version 1.3.0.

Also can I proceed with closing this PR?